onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -3793,6 +3793,75 @@ TEST_F(GraphTransformationTests, FastGeluFusionWithCastsTest3) {
   ASSERT_TRUE(op_to_count["com.microsoft.FastGelu"] == 1);
 }
 
+TEST_F(GraphTransformationTests, ReshapeFusionOpsetTest) {
```

Comment:
can you add comments explaining how the graph looks like? that would make it easy to understand the tests.

Reply:
done